### DISCUSSION TITLE
Documentation refers to old features?

### DISCUSSION THREAD

**Discussion by leepowelldev** [react-hook-form/react-hook-form#9850] [General] [answered]
The default documentation for https://react-hook-form.com/api/useform seems to refer to old features that are no longer supported in v7 such as `values` option on `useForm` - to see the correct docs I have to visit the legacy sub-domain at https://legacy.react-hook-form.com/api/useform and select v7 - seems a bit odd.

**Comment by bluebill1049** [comment, marked as answer]
`values` is new feature, so you can use with useSWR or useQquery to update form values.
